tacs/problems/base.py

```diff
@@ -1070,3 +1070,69 @@ def _addPressureFromPLOAD4(self, auxElems, loadInfo, scale=1.0):
                 self._addTractionToElements(
                     auxElems, elemID, trac, faceIndex, nastranOrdering=True
                 )
+
+    def write_sensitivity_file(self, evalFuncs, sens_file, tacs_aim=None):
+        """
+        write an ESP/CAPS .sens file from the tacs aim
```

Comment:
Use consistent docstring format (define input/outputs) see [here](https://github.com/smdogroup/tacs/blob/master/tacs/pytacs.py#L688-L732) for example